modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/RequestValidator.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.modelservice.web;
+
+import com.expedia.adaptivealerting.modelservice.model.Expression;
+import com.expedia.adaptivealerting.modelservice.model.Operand;
+import com.expedia.adaptivealerting.modelservice.model.Operator;
+import com.expedia.adaptivealerting.modelservice.model.Detector;
+import com.expedia.adaptivealerting.modelservice.model.User;
+import com.expedia.adaptivealerting.modelservice.dao.es.DetectorMappingEntity;
+import lombok.Generated;
+import org.springframework.stereotype.Component;
+import org.springframework.util.Assert;
+import org.springframework.util.StringUtils;
+
+@Component
+@Generated //(exclude from code coverage)
```

Comment:
I don't agree with excluding this class from coverage.

Reply:
Added test cases.